packages/backend-modules/mail/lib/scheduler/archiveUnsubscribed.js

```diff
@@ -9,21 +9,26 @@ module.exports = async (dryRun = false) => {
   checkEnv([
     'MAILCHIMP_ONBOARDING_AUDIENCE_ID',
     'MAILCHIMP_MARKETING_AUDIENCE_ID',
+    'MAILCHIMP_PROBELESEN_AUDIENCE_ID',
   ])
-  const { MAILCHIMP_ONBOARDING_AUDIENCE_ID, MAILCHIMP_MARKETING_AUDIENCE_ID } =
-    process.env
+  const {
+    MAILCHIMP_ONBOARDING_AUDIENCE_ID,
+    MAILCHIMP_MARKETING_AUDIENCE_ID,
+    MAILCHIMP_PROBELESEN_AUDIENCE_ID,
+  } = process.env
 
   const audiencesToArchiveUnsubscribed = [
     MAILCHIMP_ONBOARDING_AUDIENCE_ID,
     MAILCHIMP_MARKETING_AUDIENCE_ID,
+    MAILCHIMP_PROBELESEN_AUDIENCE_ID,
   ]
 
   audiencesToArchiveUnsubscribed.forEach((audienceId) => {
     archiveUnsubscribed(dryRun, audienceId)
   })
 }
 
-const archiveUnsubscribed = async (dryRun = false, audienceId) => {
+const archiveUnsubscribed = async (dryRun, audienceId) => {
```

Comment:
a) Why did the dryRun default value change?
b) Maybe this is how it's done elsewhere too but generally I'd prefer an object as function argument with named keys, i.e. `({ dryRun, audienceId }) =>`, because on the caller side you don't need to figure out the order of arguments.

Reply:
a) According to sonar it's not good practice to have a default value before a non-default value, and the dryRun parameter with default value is already passed down from the function above
b) good point :)